Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1016,21 +1017,29 @@ void findChiralSets(const ROMol &mol, DistGeom::VECT_CHIRALSET &chiralCenters,
         // we should at least have 3 though
         CHECK_INVARIANT(nbrs.size() >= 3, "Cannot be a chiral center");
 
+        double volLowerBound = 5.0;
+        double volUpperBound = 100.0;
+
         if (nbrs.size() < 4) {
+          // we get lower volumes if there are three neighbors,
+          //  this was github #5883
+          volLowerBound = 2.0;
           nbrs.insert(nbrs.end(), atom->getIdx());
         }
 
         // now create a chiral set and set the upper and lower bound on the
         // volume
         if (chiralType == Atom::CHI_TETRAHEDRAL_CCW) {
           // positive chiral volume
-          auto *cset = new DistGeom::ChiralSet(atom->getIdx(), nbrs[0], nbrs[1],
-                                               nbrs[2], nbrs[3], 5.0, 100.0);
+          auto *cset =
```

Comment:
I'm assuming here that CHI_TETRAHEDRAL_CCW assumes nbrs.size() == 4 but perhaps this should be a runtime check in case someone futzed with the molecule.

Reply:
(This isn't related to this PR directly)

Reply:
You can have tetrahedral stereo with only three neighbors. There's a test a few lines higher up that makes sure that `nbrs` has four members: https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/DistGeomHelpers/Embedder.cpp#L1019